pandas/core/arrays/categorical.py

```diff
@@ -1140,6 +1144,8 @@ def remove_categories(self, removals, inplace=False):
            Whether or not to remove the categories inplace or return a copy of
            this categorical with removed categories.
 
+           .. deprecated:: 1.2.0
```

Comment:
1.3.0 now

Reply:
Nice catch, thanks. I updated the version here and fixed conflicts with the master